frontend/components/dashboard/DashboardRenameModal.vue

```diff
@@ -56,6 +58,12 @@ const rename = async () => {
 </template>
 
 <style lang="scss" scoped>
+@use "~/assets/css/utils.scss";
+
+:global(.dashboard-rename-modal-header .p-dialog-title) {
```

Comment:
As I do not know it:
Is it sure that this selector will be removed from global rules by Vue when this dialog component is unmounted ?

Reply:
yes that should be the case, but just to be sure it has a distinct name (`dashboard-rename-modal-header`) so it should never bother any other styles.